Blogpost for the first episode of the podcast (#107)
publiccodenet/blog: _posts/2020-09-23-episode-one-of-lets-talk-about-public-code.md

---
title: "Episode one of Let's talk about public code!"
date: 2020-09-23
author: Jan Ainali
type: blogpost
excerpt: The first episode of our podcast is available
categories:
  - news
---

# Episode one of Let's talk about public code!

Recently we [shared the news](https://blog.publiccode.net/news/2020/09/01/on-air-next-our-podcast.html) about our upcoming podcast *Let's talk about public code*. Now when the premier is over we want to share all the links so that you can watch it again if you missed the livestream or find the audio version in your podcast player. We also want to thank Dr. Arnau Monterde, coordinator from the popular project [Decidim](https://decidim.org/), that honoured us with a lovely chat in this first episode.

## Where can I find the video?

The video is available in a number of places:

- [Youtube](https://www.youtube.com/watch?v=MFAb8QaDXFY)
- [Periscope](https://www.pscp.tv/w/1nAKEAQlonkKL)
- [Facebook](https://www.facebook.com/publiccodenet/videos/643877099845339/)

[![Screenshot of the podcast]({{site.url}}/assets/screenshot-podcast.png)](https://www.youtube.com/watch?v=MFAb8QaDXFY)

## Where can I find the podcast?

> Do you prefer just listen to the audio? Then you're in for a treat. We have published a polished version of the interview at [podcast.publiccode.net](https://podcast.publiccode.net/). It is also listed in the following platforms so it's a good chance you'll find us directly in your favourite podcast player if it's something else than Podbean where we are hosted:

Reviewer: ```suggestion
Do you prefer to listen to just the audio? Then you're in for a treat. We have published a polished version of the interview at [podcast.publiccode.net](https://podcast.publiccode.net/). 
It is also listed in the following platforms so there's a good chance you'll find us directly on your favourite podcast player:
```

Do we need to mention that we use Podbean? If it's important, should we include a link to it?

Author: No, it's not important at all, good catch!
